src/Components/Utils/SnackBar/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+import React from 'react';
+import MatieralSnackBar from 'material-ui/Snackbar';
+import { connect } from 'react-redux';
+import type { Connector } from 'react-redux';
+import { clearError as reduxClearError } from '../../../Stores/Common/actions';
+import type { Dispatch, State } from '../../../types';
+
+type OwnProps = {};
+
+type ReduxProps = {
+  error: ?string,
+  clearError: () => void,
+};
+
+type Props = OwnProps & ReduxProps;
+
+class SnackBar extends React.Component {
+  props: Props;
+
+  render() {
+    const { error, clearError } = this.props;
```

Comment:
What if we want to show success messages?

Reply:
Good point.
Rename this to `message` ?
`text`?
`alertText` ?

Reply:
yeah maybe "show" as true/false and "message" as a string?

Reply:
I figured the `show` state could just be `null`

Flow catches that pretty nicely